serniki/sernik-codilime.md

```diff
@@ -0,0 +1,79 @@
+Sernik nazwany na cześć firmy [CodiLime](https://www.codilime.com/), w której
+pracuję, i która finansuje badania nad tym przepisem (zjedli już kilkanaście
+sztuk ^^)
+
+## Składniki na dużą tortownicę (26-29 cm średnicy)
+
+- 800-1000 g sera (patrz komentarz poniżej)
+- 200-400 g śmietany lub śmietanki (patrz komentarz poniżej)
+- 5 dużych jajek (klasa wagowa 63-73 g)
+- 1/2 kostki Kasi lub masła
+- 1 szkl. drobnego cukru do wypieków (220g)
+- łyżeczka cukru waniliowego
+- 1 budyń waniliowy bez cukru (40g)
+- paczka okrągłych biszkoptów (ok. 100-120g, zależnie od rozmiaru tortownicy)
+- Opcjonalnie dodatek aromatu cytrynowego albo pomarańczowego
+
+*Uwaga: rodzaj użytego sera ma duże znaczenie. Najbardziej lubię używać twarogu
+zmielonego z Wielunia (ew. Piątnicy), który jest dość gęsty i słodki - wtedy
+żeby dodać mu charakteru i rozrzedzić go dodaję kwaśnej śmietany (w proporcji
+3:1). Jeśli Twój ser jest sam z siebie dość kwaśny, użyj słodkiej śmietanki, a
```

Comment:
Oczywiście, sierotek należy unikać :-) Ale to jest markdown i GitHub umie go [ładnie wyświetlać](https://github.com/jan-warchol/pan-deser/blob/sernik-codilime/serniki/sernik-codilime.md), łącząc linie w akapity i tak dalej. Wydaje mi się, że to wystarczy, a w kodzie może zostać tak jak jest - użytkownicy będą raczej korzystać z wyrenderowanej wersji. Tym bardziej, że pisząc kod używam automatycznego łamania linii w vimie, a ono nie umie unikać sierotek (a nie chciałbym poprawiać po nich ręcznie za każdym razem). Nie sądzę zresztą żeby jakikolwiek edytor kodu wiedział co to sierotki. Jak myślisz?

Pozostałymi rzeczami zajmę się po pracy :-)